templates/app/team.phtml

```diff
@@ -10,51 +10,51 @@ $this->headTitle($title);
         <li>
-            <img src="/images/team/lee.png" />
-            <h2>Lee Boynton</h2>
+            <img src="/images/team/oli.jpg" />
+            <h2>Oli Northam</h2>
             <p>
-                Lee is a "full stack" developer based in Portsmouth, UK who is always interested in experimenting with
-                new technologies. This ranges from the latest PHP frameworks, to front-end JavaScript libraries and to
-                Linux server management tools. Other interests include security, code quality and automation.
+                Oli is a developer from Portsmouth, currently working with innovative digital design and development
+                firm Si digital. He's a fan of 90's music, and Dance Dance Revolution.
             </p>
             <p>
-                <a href="http://lboynton.com/">lboynton.com</a> | <a href="https://twitter.com/leeboynton">@leeboynton</a>
+                <a href="http://oli.me/">oli.me</a> | <a href="https://twitter.com/mrnorm">@mrnorm</a>
             </p>
         </li>
         <li>
-            <img src="/images/team/oli.jpg" />
-            <h2>Oli Northam</h2>
+            <img src="/images/team/lee.png" />
+            <h2>Lee Boynton</h2>
             <p>
-                Oli is a developer from Portsmouth, currently working with innovative digital design and development
-                firm Si digital. He's a fan of 90's music, and Dance Dance Revolution.
+                Lee is a "full stack" developer based in Portsmouth, UK who is always interested in experimenting with
+                new technologies. This ranges from the latest PHP frameworks, to front-end JavaScript libraries and to
+                Linux server management tools. Other interests include security, code quality and automation.
             </p>
             <p>
-                <a href="http://oli.me/">oli.me</a> | <a href="https://twitter.com/mrnorm">@mrnorm</a>
+                <a href="http://lboynton.com/">lboynton.com</a> | <a href="https://twitter.com/leeboynton">@leeboynton</a>
             </p>
         </li>
         <li>
-            <img src="/images/team/jez.jpg" />
-            <h2>Jez Emery</h2>
+            <img src="/images/team/james.png" />
+            <h2>James Titcumb</h2>
             <p>
-                Jez works at Strawberrysoup working with their clients and infrastructure. Jez has been an active member of the PHP
-                Hampshire community for a few years now, and his relentless enthusiasm for building the community has
-                resulted in him becoming one of the team.
+                James is the founder of the UK based PHP Hampshire user group and the PHP South Coast Conference. He's
+                also a Zend Certified Engineer and consultant at Roave. During his downtime, he continues to run the PHP
+                Hampshire user group and the conference, and keeps up with active contributions to various open source
+                projects.
```

Comment:
Perhaps the text should be changed to "continues to support" rather than "continues to run"